### PR TITLE
Update Send Data page - add links to Kubernetes Solution

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
## Purpose of this pull request

Update Send Data page - add links to Kubernetes Solution

Issue number:  https://jira.kumoroku.com/jira/browse/SUMO-215431

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+id: kubernetes
+title: Kubernetes
+description: Sumo Logic offers a comprehensive Kubernetes Solution, however, if you only want to know how to set up collection you can follow our deployment guide referenced in this document.
+---
+
+The [Deployment Guide](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/README.md) in our sumologic-kubernetes-collection GitHub repository has detailed instructions on how to collect Kubernetes logs, metrics, and events; enrich them with deployment, pod, and service level metadata; and send them to Sumo Logic.
+
+This guide is the same used for setting up collection for our [comprehensive Kubernetes solution](/docs/observability/kubernetes).
```
Comment (JainM6) [inline docs/send-data/kubernetes.md:9]:
```suggestion
This is the same guide used for setting up a collection for our [comprehensive Kubernetes solution](/docs/observability/kubernetes). 
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Looks good, just a minor correction.

**Review by JainM6 — CHANGES_REQUESTED** [review]
This path needs to be fixed - docs/send-data/collect-from-other-data-sources/kubernetes/. I see that this file exists under the collect-from-other-data-sources, but you have removed it.

**Comment by kkujawa-sumo** [comment]
>This path needs to be fixed - docs/send-data/collect-from-other-data-sources/kubernetes/. I see that this file exists under the collect-from-other-data-sources, but you have removed it.

@JainM6 My intention is to move the file from `docs/send-data/collect-from-other-data-sources/kubernetes.md` to `docs/send-data/kubernetes.md`. I see that there are other references in docs to `send-data/collect-from-other-data-sources/kubernetes` which are not fixed now. I'll fix them.

**Review by kimsauce — CHANGES_REQUESTED** [review]
Hi @kkujawa-sumo , I don't think Kubernetes belongs at the Send Data root level. This is reserved for high-level collector categories, not sources. Is there another Send Data section this could be placed under? 

For such a big move, we'd also need to get buy-in from collection PMs.

Additionally, when you move a doc, you need to [follow all of these steps](https://help.sumologic.com/docs/contributing/remove-doc/).

**Comment by kkujawa-sumo** [comment]
@kimsauce what do you think about correction? This will only add the link to Kubernetes page in sidecar menu and the link on Send Data page. Is it better?

File: docs/send-data/index.md [hunk 2]
```diff
@@ -34,31 +34,37 @@ Learn how to send data to Sumo Logic and get started with our observability and
   <p>Set up a Hosted Collector to collect data directly from cloud sources like AWS, GCP, and more without installing an agent.</p>
   </div>
 </div>
-  <div className="box smallbox4 card">
+<div className="box smallbox_4 card">
+  <div className="container">
+  <a href="/docs/send-data/kubernetes"><img src={useBaseUrl('img/icons/operations/kubernetes.png')} alt="icon" width="40"/><h4>Kubernetes</h4></a>
```
Comment (kimsauce) [inline docs/send-data/index.md:39]:
```suggestion
  <a href="/docs/send-data/collect-from-other-data-sources/kubernetes"><img src={useBaseUrl('img/icons/operations/kubernetes.png')} alt="icon" width="40"/><h4>Kubernetes</h4></a>
```

**Comment by kimsauce** [comment]
> @kimsauce what do you think about correction? This will only add the link to Kubernetes page in sidecar menu and the link on Send Data page. Is it better?

That's perfect - great idea, thank you! Just approved.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**